diff -ru cfm/cfm-annotate/CMakeLists.txt cfm-install-patch/cfm-annotate/CMakeLists.txt
--- cfm/cfm-annotate/CMakeLists.txt	2016-06-12 23:17:57.000000000 +0200
+++ cfm-install-patch/cfm-annotate/CMakeLists.txt	2016-06-12 23:02:46.000000000 +0200
@@ -13,10 +13,10 @@
 target_link_libraries ( cfm-annotate cfm-code ${REQUIRED_LIBS} )
 
 install ( TARGETS cfm-annotate 
-          DESTINATION ${CFM_OUTPUT_DIR} )
+          DESTINATION bin )
           
 if (WIN32)
     install ( FILES ${LPSOLVE_INCLUDE_DIR}/lpsolve55.dll
               DESTINATION ${CFM_OUTPUT_DIR} )
 endif (WIN32)              
-          
\ No newline at end of file
+          
diff -ru cfm/cfm-code/CMakeLists.txt cfm-install-patch/cfm-code/CMakeLists.txt
--- cfm/cfm-code/CMakeLists.txt	2016-06-12 23:17:57.000000000 +0200
+++ cfm-install-patch/cfm-code/CMakeLists.txt	2016-06-12 23:02:22.000000000 +0200
@@ -54,4 +54,4 @@
 target_link_libraries ( cfm-code ${REQUIRED_LIBS} )
 
 install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/ISOTOPE.DAT
-          DESTINATION ${CFM_OUTPUT_DIR} )    
\ No newline at end of file
+          DESTINATION share )    
diff -ru cfm/cfm-id/CMakeLists.txt cfm-install-patch/cfm-id/CMakeLists.txt
--- cfm/cfm-id/CMakeLists.txt	2016-06-12 23:17:56.000000000 +0200
+++ cfm-install-patch/cfm-id/CMakeLists.txt	2016-06-12 23:03:52.000000000 +0200
@@ -13,10 +13,10 @@
 target_link_libraries ( cfm-id cfm-code ${REQUIRED_LIBS} )
 
 install ( TARGETS cfm-id 
-          DESTINATION ${CFM_OUTPUT_DIR} )
+          DESTINATION bin )
           
 if (WIN32)
     install ( FILES ${LPSOLVE_INCLUDE_DIR}/lpsolve55.dll
               DESTINATION ${CFM_OUTPUT_DIR} )
 endif (WIN32)              
-          
\ No newline at end of file
+          
diff -ru cfm/cfm-id-precomputed/CMakeLists.txt cfm-install-patch/cfm-id-precomputed/CMakeLists.txt
--- cfm/cfm-id-precomputed/CMakeLists.txt	2016-06-12 23:17:57.000000000 +0200
+++ cfm-install-patch/cfm-id-precomputed/CMakeLists.txt	2016-06-12 23:03:31.000000000 +0200
@@ -13,10 +13,10 @@
 target_link_libraries ( cfm-id-precomputed cfm-code ${REQUIRED_LIBS} )
 
 install ( TARGETS cfm-id-precomputed 
-          DESTINATION ${CFM_OUTPUT_DIR} )
+          DESTINATION bin )
           
 if (WIN32)
     install ( FILES ${LPSOLVE_INCLUDE_DIR}/lpsolve55.dll
               DESTINATION ${CFM_OUTPUT_DIR} )
 endif (WIN32)              
-          
\ No newline at end of file
+          
diff -ru cfm/cfm-predict/CMakeLists.txt cfm-install-patch/cfm-predict/CMakeLists.txt
--- cfm/cfm-predict/CMakeLists.txt	2016-06-12 23:17:57.000000000 +0200
+++ cfm-install-patch/cfm-predict/CMakeLists.txt	2016-06-12 23:06:29.000000000 +0200
@@ -13,10 +13,10 @@
 target_link_libraries ( cfm-predict cfm-code ${REQUIRED_LIBS} )
 
 install ( TARGETS cfm-predict
-          DESTINATION ${CFM_OUTPUT_DIR} )
+          DESTINATION bin )
           
 if (WIN32)
     install ( FILES ${LPSOLVE_INCLUDE_DIR}/lpsolve55.dll
               DESTINATION ${CFM_OUTPUT_DIR} )
 endif (WIN32)              
-          
\ No newline at end of file
+          
diff -ru cfm/cfm-test/CMakeLists.txt cfm-install-patch/cfm-test/CMakeLists.txt
--- cfm/cfm-test/CMakeLists.txt	2016-06-12 23:17:57.000000000 +0200
+++ cfm-install-patch/cfm-test/CMakeLists.txt	2016-06-12 23:13:00.000000000 +0200
@@ -25,10 +25,10 @@
 target_link_libraries ( cfm-test cfm-code ${REQUIRED_LIBS} )
 
 install ( TARGETS cfm-test 
-          DESTINATION ${CFM_OUTPUT_DIR} )
+          DESTINATION bin )
           
 install ( DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/tests/test_data
-          DESTINATION ${CFM_OUTPUT_DIR}/tests )          
+          DESTINATION bin )          
+          
           
           
-          
\ No newline at end of file
diff -ru cfm/cfm-train/CMakeLists.txt cfm-install-patch/cfm-train/CMakeLists.txt
--- cfm/cfm-train/CMakeLists.txt	2016-06-12 23:17:56.000000000 +0200
+++ cfm-install-patch/cfm-train/CMakeLists.txt	2016-06-12 23:13:10.000000000 +0200
@@ -13,7 +13,7 @@
 target_link_libraries ( cfm-train cfm-code ${REQUIRED_LIBS} )
 
 install ( TARGETS cfm-train 
-          DESTINATION ${CFM_OUTPUT_DIR} )
+          DESTINATION bin )
+          
           
           
-          
\ No newline at end of file
diff -ru cfm/fraggraph-gen/CMakeLists.txt cfm-install-patch/fraggraph-gen/CMakeLists.txt
--- cfm/fraggraph-gen/CMakeLists.txt	2016-06-12 23:17:57.000000000 +0200
+++ cfm-install-patch/fraggraph-gen/CMakeLists.txt	2016-06-12 23:13:24.000000000 +0200
@@ -13,10 +13,10 @@
 target_link_libraries ( fraggraph-gen cfm-code ${REQUIRED_LIBS} )
 
 install ( TARGETS fraggraph-gen 
-          DESTINATION ${CFM_OUTPUT_DIR} )
+          DESTINATION bin )
           
 if (WIN32)
     install ( FILES ${LPSOLVE_INCLUDE_DIR}/lpsolve55.dll
               DESTINATION ${CFM_OUTPUT_DIR} )
 endif (WIN32)              
-          
\ No newline at end of file
+          
